app/controllers/mixins/breadcrumbs_mixin.rb

```diff
@@ -52,7 +52,7 @@ def data_for_breadcrumbs
     end
 
     def action_breadcrumb?
-      @sb["action"] && @right_cell_text
+      @sb["action"] && @right_cell_text && params[:button] != 'cancel' && @sb[:action] != 'tree_select'
```

Comment:
:fire: :rotating_light: :skull: :biohazard: you're introducing potential 🦛🦛 technical debt 🦛🦛 here :disappointed: